.github/workflows/lint-test.yml

```diff
@@ -7,9 +7,6 @@ on:
   push:
     branches: [main]
   pull_request:
-  repository_dispatch:
-    types: [test-agoric-sdk]
-  # FIXME: Deprecated build from schedule.
   schedule:
     - cron: "0 12 * * *"
```

Comment:
does this run a nightly build/test even if nothing has changed?

Reply:
I think so, according to https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07. Maybe we shouldn't?

Reply:
I'll just merge it for now and we can take it out later if desired